curriculum/challenges/english/01-responsive-web-design/applied-accessibility/add-a-text-alternative-to-images-for-visually-impaired-accessibility.english.md

```diff
@@ -8,15 +8,15 @@ guideUrl: 'https://www.freecodecamp.org/guide/certificates/add-alt-text-to-an-im
 
 ## Description
 <section id='description'>
-It's likely you've seen an <code>alt</code> attribute on an <code>img</code> tag in other challenges. <code>Alt</code> text describes the content of the image and provides a text-alternative. This helps in case the image fails to load or can't be seen by a user. It's also used by search engines to understand what an image contains to include it in search results. Here's an example:
+It's likely that you've seen an <code>alt</code> attribute on an <code>img</code> tag in other challenges. <code>Alt</code> text describes the content of the image and provides a text-alternative for it. This helps in cases where the image fails to load or can't be seen by a user. It's also used by search engines to understand what an image contains to include it in its search results. Here's an example:
```

Comment:
I think the word `its` can be left out of the phrase:
> ... contains to include it in its search results.

I believe the sentence will read find as it was originally:
> It's also used by search engines to understand what an image contains to include it in search results.